core/storage/user/gae_models.py

```diff
@@ -59,6 +63,14 @@ class UserSettingsModel(base_models.BaseModel):
     username = ndb.StringProperty(indexed=True)
     # Normalized username to use for duplicate-username queries. May be None.
     normalized_username = ndb.StringProperty(indexed=True)
+    # Display name for users.
+    name = ndb.StringProperty(default=None, indexed=True)
+    # List of profile user ids associated with this user. None for profiles.
+    profiles_list = ndb.StringProperty(default=None, repeated=True)
+    # Pin assocaited with the user profile.
```

Comment:
Here and elsewhere, please check your spelling.

Explain what the PIN is used for.